packages/eslint-config/README.md

```diff
@@ -0,0 +1,35 @@
+# Wire
+
+This repository is part of the source code of Wire. You can find more information at [wire.com](https://wire.com) or by contacting opensource@wire.com.
+
+You can find the published source code at [github.com/wireapp](https://github.com/wireapp).
+
+For licensing information, see the attached LICENSE file and the list of third-party licenses at [wire.com/legal/licenses/](https://wire.com/legal/licenses/).
+
+# eslint-config
+
+> Wire's [ESLint](https://eslint.org/docs/developer-guide/shareable-configs) config.
+
+## Usage
+
+**Install**:
+
+```bash
+$ yarn add --dev @wireapp/eslint-config \
```

Comment:
Let's remove the `$ ` because it will prevent easy copy & paste.

Reply:
btw: This command will not work on Windows.

Reply:
Added two options after all.. all in one line is messy.. we have space!